### ISSUE TITLE
｢rmdir｣ documentation is a bit off

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/doc#852]
``` perl6
sub    rmdir(IO $dir --> Bool)
method rmdir(IO::Path:D: --> Bool)
```

First of all, the sub form does not really return Bool:

```
 perl6 -e 'say rmdir ‘deletable’' # OUTPUT: []
```

And also it allows many dirs to be passed ([source](https://github.com/rakudo/rakudo/blob/d927228a1e621eaa7342447f9695aa245ee092ca/src/core/io_operators.pm#L232)):

``` perl6
sub rmdir(*@filenames, :$SPEC = $*SPEC, :$CWD = $*CWD) {
    my @ok;
    for @filenames -> $file {
        @ok.push($file) if $file.IO(:$SPEC,:$CWD).rmdir;
    }
    @ok;
#    @filenames.grep( *.IO(:$SPEC,:$CWD).rmdir ).eager;
}
```

Also, it does not mention how to wipe a whole directory with all of its contents, I was looking at the documentation of `rmdir` for exactly this purpose. As of now, it looks like there is [no easy way](http://irclog.perlgeek.de/perl6/2016-08-22#i_13067165), but it would be great if we could cover some alternative solutions.

**Comment by AlexDaniel** [comment]
One of the possible solutions is ``rmtree`` from [File::Directory::Tree](https://github.com/labster/p6-file-directory-tree).

**Comment by zoffixznet** [comment]
This was resolved by 8de94c9 and IO grant
